source/core/src/main/com/csse3200/game/areas/ForestGameArea.java

```diff
@@ -84,8 +87,8 @@ public void create() {
     spawnTerrain();
     spawnTrees();
     player = spawnPlayer();
-    //spawnGhosts();
-    //spawnGhostKing();
+//    spawnGhosts();
```

Comment:
Needs to be deleted if they are not in our game

Reply:
I can delete it but if you still do check main the ghost code is there 
So no changes were made because of that